prep-javascript-objects/main.js

```diff
@@ -0,0 +1,18 @@
+var person = {
+  firstName: 'Josh',
+  lastName: 'Tenny',
+  hobby: 'Jujitsu'
+
+};
+
+console.log(person);
+
+var fullName = ' My full name is ' + 'Joshua ' + 'Tenny.';
+console.log(fullName);
```

Comment:
ok, great, this is almost perfect, but we can use the firstName and lastName properties of the person object to construct the fullName

```
var fullName = ' My full name is ' + person.firstName + ' ' + person.lastName;
//                                                       ^ this is to make sure we get a space in there
```

Reply:
![image](https://user-images.githubusercontent.com/31152745/203233799-9d975022-9e7d-4379-b1a8-72a17c634271.png)